processTimeTrackingEntries.py

```diff
@@ -293,14 +293,16 @@ def main():
                 'created as worklog in JIRA and subsequently tagged in Toggl'.format(
                     str(time_entry['id']), time_entry['description']))
 
+    # when last Toggl time entry is running (duration is negative), all entries should be skipped.
+    if (time_entry["duration"] < 0):
+        _logger.error("Toggl is still running! No time entries will be processed. "
+                        "Stop the current time entry and execute this script again!")
+        return
+
     for key, grouped_time_entry in grouped_time_entries.items():
         if (len(grouped_time_entry["time_entries"]) > 0):
             start_time = min(time_entry["start_time"] for time_entry in grouped_time_entry["time_entries"])
 
-            # when Toggl is running (duration is negative), the entry should be skipped.
-            if (time_entry["duration"] < 0):
-                continue
-
```

Comment:
Sorry - den PR hatte ich nicht mehr auf dem Zettel.

Ich kann nicht ganz nachvollziehen, warum das Skript komplett abbrechen soll wenn ein Eintrag noch läuft. Kannst du den Teil bitte in einen eigenen PR rausziehen? Dann kann ich den Bug mit dem Dateinamen mergen und das andere können wir uns dann ansehen.